x/execution/internal/keeper/keeper.go

```diff
@@ -221,14 +225,30 @@ func (k *Keeper) distributePriceShares(ctx sdk.Context, execAddress, runnerAddre
 	}
 
 	runnerCoins, _ := sdk.NewDecCoinsFromCoins(coins...).MulDecTruncate(runnerShare).TruncateDecimal()
-	serviceCoins := coins.Sub(runnerCoins)
-	// TODO: add emitters share
+	emittersCoins, _ := sdk.NewDecCoinsFromCoins(coins...).MulDecTruncate(emittersShare).TruncateDecimal()
+	serviceCoins := coins.Sub(runnerCoins).Sub(emittersCoins)
+
+	var voters []sdk.AccAddress
+	var emitterCoins sdk.Coins
+	if store.Has(voteKey(execHash)) {
```

Comment:
We shouldn't do this test, if there is no vote on the store it means we have a serious issue before and we should either panic or at least return an error and not just ignore it I think. Or we could calculate everything and at the end assert that the result of emitters + runner + service == balance of execution.

Reply:
This is related to a question if we should put a exec creator as an voter into the list. If yes, then this check shouldn't be here, otherwise it's should be :)